src/Symfony/Bridge/Doctrine/Tests/ArgumentResolver/EntityValueResolverTest.php

```diff
@@ -415,9 +415,16 @@ private function createRegistry(?ObjectManager $manager = null): ManagerRegistry
             ->method('getManagerForClass')
             ->willReturn($manager);
 
-        $registry->expects($this->any())
-            ->method('getManager')
-            ->willReturn($manager);
+        if (method_exists(ObjectManager::class, 'isUninitializedObject')) {
```

Comment:
This should check `$manager === null`. We don't want to throw when we have an instance.

And I suggest applying this condition even for older versions of doctrine/persistence so that the mock reflects the actual behavior of the class. AbstractManagerRegistry was already throwing an exception in persistence 2.x (I did not check the behavior in 1.x)

Reply:
I know. That's exactly why I did not check 1.x to see whether our mocks were already wrong back in time or whether they kept emulating the 1.x behavior, because distinguishing those 2 cases does not matter.